build/bazel/remote/execution/v2/remote_execution.proto

```diff
@@ -1997,6 +2018,11 @@ message CacheCapabilities {
   // [BatchUpdateBlobs][build.bazel.remote.execution.v2.ContentAddressableStorage.BatchUpdateBlobs]
   // requests.
   repeated Compressor.Value supported_batch_update_compressors = 7;
+
+  // The digest of the
+  // [ZstdDictionaryRegistry][build.bazel.remote.execution.v2.ZstdDictionaryRegistry]
+  // that contains all the dictionaries supported by the remote cache.
+  Digest zstd_dictionary_registry = 8;
```

Comment:
Oh this is brilliant!
QualifiedDigest is exactly what I wanted.

Will add this into the next iteration.

Reply:
IMO, this is not the right time to introduce `QualifiedDigest`. For better or for worse, the existing convention is to make `DigestFunction.Value` a sibling field of `Digest`. Without concrete plans to migrate the other existing usages, I think this will add unnecessary inconsistency to the API, introduce a new "busywork" task of everyone having to migrate from Digest => QualifiedDigest, and cause confusion for both clients and servers (e.g. "why do we sometimes use QualifiedDigest and other times we use a regular Digest?").

Reply:
NVM, I see that this is outdated. How embarrassing :sweat_smile: